dts/arm/nxp/nxp_rt5xx_common.dtsi

```diff
@@ -721,6 +722,16 @@
 			status = "disabled";
 		};
 	};
+
+	inputmux: inputmux@26000 {
```

Comment:
You include the rt5xx.h file in this commit but don't use any of the definitions inside it, is there not suppose to be a default inputmux value?

Reply:
it's probably because the board level will include the soc file and thereby get the definitions transitively